app/helpers/local_time_helper.rb

```diff
@@ -67,4 +53,33 @@ def extract_options_and_value(options, value_key = nil)
         [ {}, options ]
       end
     end
+
+    def find_time_format(format)
+      if format.is_a?(Symbol)
+        find_time_format_by_name(format)
+      else
+        [format.presence || LocalTime.default_time_format, nil]
+      end
+    end
+
+    def find_time_format_by_name(format)
```

Comment:
What do you think about naming this `find_time_formats_by_name` to make it more obvious that it nows returns two formats?